std/variant.d

```diff
@@ -285,7 +285,14 @@ private:
         static bool tryPutting(A* src, TypeInfo targetType, void* target)
         {
             alias UA = Unqual!A;
-            alias MutaTypes = AliasSeq!(UA, ImplicitConversionTargets!UA);
+            static if(isStaticArray!A && is(typeof(UA.init[0])))
```

Comment:
take care to the D style, here a space misses between the keyword and the left paren. `if (`

Reply:
Thanks.